### ISSUE TITLE
Warning: PHP Startup: Unable to load dynamic library 'C:\xampp\php\ext\php_fann.dll'

### ISSUE THREAD

**Issue by paul-bumperactive** [bukka/php-fann#49]
Running PHP on Windows 10 Pro.
PHP Version: **PHP 8.2.12  Thread safe version**

ph-_fan.dll version: **Tried both 8.1 and 7.4**
php_fann-1.2.0rc2-8.1-ts-vs16-x64
php_fann-1.1.1-7.4-ts-vc15-x64

**PHP.ini  entries**:
extension_dir="C:\xampp\php\ext"
extension="C:\xampp\php\ext\php_fann.dll"
"doublefann.dll" placed in C:/xampp/php which is included in the system PATH variable.

Directory listing:
dir C:\xampp\php\ext\php_fann.dll
 Volume in drive C is OS
 Volume Serial Number is A88E-9D2F
 Directory of C:\xampp\php\ext
01/19/2024**  11:37 AM            91,648 php_fann.dll
               1 File(s)         91,648 bytes
               0 Dir(s)  197,248,303,104 bytes free

**Error when starting PHP**:
PHP Warning:  PHP Startup: **Unable to load dynamic library 'C:\xampp\php\ext\php_fann.dll'** (tried: C:\xampp\php\ext\php_fann.dll (The
 specified module could not be found), C:\xampp\php\ext\php_C:\xampp\php\ext\php_fann.dll.dll (The specified module could not be found)) in Unknown on line 0

**Comment by paul-bumperactive** [comment]
I just downgraded to PHP 8.1.25 and php_fann loaded.

Are the dlls not backwards compatible with higher PHP versions?
Does this mean that every time a new PHP version is released, we are dead until a corresponding fann is released or is this just a facet of 8.2 PHP?

**Comment by bukka** [comment]
Yeah pecl libs are no longer building. I will have to integrate GitHub pipeline action to do it and provide dll's from here.

**Comment by macintoshplus** [comment]
I am pleased to announce that this extension is now compiled for Windows by the PHP Extension Repository site.

You can download the DLLs on the extension page: https://phpext.phptools.online/extension/tools-and-utilities/fann-283

If you have any questions, I am available to discuss them.
